openc3/lib/openc3/models/scope_model.rb

```diff
@@ -74,13 +74,13 @@ def self.all(scope: nil)
       super(PRIMARY_KEY)
     end
 
-    def self.from_json(json)
+    def self.from_json(json, scope: nil)
```

Comment:
scope is unused ... why is this necessary?

Reply:
It matches the syntax for the base class.   Otherwise more of the default methods have to be overriden.